### PR TITLE
[ADD] purchase: RFQ 

### PR DESCRIPTION
**Pull request by dikd-odoo**
First gen documentation on creating RFQ in the Purchase app.

### CODE REVIEW

**Comment by StraubCreative** [comment]
Hi @Felicious this PR on RFQ basics is ready for your peer review!

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:115]: While it's definitely cool that we can send emails through Odoo, this doc is about the process of creating RFQs and |POs|. To keep the focus on that, I'm making a suggestion to gloss over these emailing features.

```suggestion
address (configured in the **Contacts** app). After crafting the desired message, click :guilabel:`Send`. Once the email has been sent, the |RFQ| moves to the
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:116]: Since it's text that we see explicitly on the UI
```suggestion
:guilabel:`RFQ Sent` stage.
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:119]:
```suggestion
:guilabel:`Print RFQ` downloads a PDF version of the |RFQ|.
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:126]: Suggestion to remove since there are basically no conditions needed to fulfill to click the Confirm Order button. (Except for entering a Vendor and Order Deadline, but you need that for an RFQ too)
```suggestion
Clicking :guilabel:`Confirm Order` directly transforms the RFQ into an active order.
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:133]: We can also elaborate that the Order Deadline field on an RFQ has turned into the Confirmation Date, displaying the date and time the user confirmed the order.

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:142]: we can also put a seealso admonition block at the end to bring attention to the next stage after receiving products, which is paying the vendors for the products they supplied us

```suggestion
   with the product information and expected arrival dates automatically populated.
   
.. seealso:
   (For Dina to do: link to this doc using the "doc" directive: https://www.odoo.com/documentation/16.0/applications/inventory_and_mrp/purchase/manage_deals/manage.html)
```

[RST cheatsheet for linking to other pieces of documentation](https://www.odoo.com/documentation/16.0/contributing/documentation/rst_cheat_sheet.html#documentation-pages-hyperlinks)

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:138]: Nitpick: the Receive Products button is a normal button, and not a smart button. (The smart button is Receipts in the image below)
```suggestion
   After an order is placed, clicking :guilabel:`Receive Products` records the reception of new products into the database.
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:99]: Content in these admonition blocks are supplementary information. Since the expected arrival field is very important, I'd take it out and have it join its friends in the paragraph above.

**Review by Felicious — CHANGES_REQUESTED** [review]
Hi @dikd-odoo ! 

Amazing job opening your first PR,*without* build errors! :blush: Very impressive :heart: 

In my review, I was strict with the accuracy of content because you're working on an essential doc that many readers will read! Most of my comments were about formatting, which is something you'll get used to the longer you write with our team :blush: I also had a few comments to make improvements on brevity and specificity of the behaviors of the Odoo software.

Overall, you did a very good job with capturing the flow and defining the various fields and menus. I can tell you read our style guidelines, as the content is pretty close to how our team structures things! Also, I'm super proud that you know about icons already! :1st_place_medal:

After you take a look at my comments, and [follow the instructions to implement suggestions from Github](https://www.odoo.com/odoo/knowledge/3132?cids=3), feel free to tag me again for a second review! Looking forward to helping you bring the doc to publication! :heart: 

P.S. let me know if you need clarification on any of my comments! Happy to elaborate :heart_hands:

**Comment by dikd-odoo** [comment]
Hi @Felicious thank you for the great feedback! I incorporated your changes and some content and formatting edits, this is ready for a second review when you have time.

Thank you also for the sanity checks whenever I think I break github

**Comment by dikd-odoo** [comment]
Hi @Felicious thank you so much for the great feedback! Incorporated your changes and made a few content/formatting changes. 

PR is ready for a second review when you have a chance. TY again!

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:156]:
```suggestion
On the new |PO|, the :guilabel:`Order Deadline` turns into :guilabel:`Confirmation Date`, which
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:159]:
```suggestion
Depending on the user's chosen configuration in the **Purchase** app settings, a *vendor bill* is
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:161]: Minor nitpick-- we typically have a space between link text ("this documentation"), and the link itself ("manage")
```suggestion
refer to :doc:`this documentation <manage>`.
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:169]: Great job on this, Dina!! :heart_eyes:

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:168]: We bold app names now (as of a couple days ago :sweat_smile: )
```suggestion
   With the **Inventory** app installed, confirming a *PO* automatically creates a receipt document,
```

**Review by Felicious — APPROVED** [review]
Thank you for implementing my suggestions so diligently, @dikd-odoo :blush: 

The doc goes into instructs well, and uses an adequate amount of images and note/tip blocks to share information. I just had minor formatting suggestions for you to implement, but after that, the doc is good for the next stage of review. Amazing job, Dina!

**Comment by dikd-odoo** [comment]
Hi @Felicious thanks for guiding me through these changes as well as the sanity check on my build! :smile:

@StraubCreative @odoo/us-doc-review Hello! This PR is ready for final review

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:155]:
```suggestion
   Odoo tracks communications on each order through the chatter of the |PO| form. This shows the
   emails sent between the user and the contact, as well as any internal notes and activities.
   Messages, notes, and activities can also be logged on chatter.
```
depending on the zoom/browser size, the chatter can appear on the right *or* the bottom, so it's always best to keep that kind of tip as generic as possible.

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:157]:
```suggestion
Once an |RFQ| is confirmed, it creates a |PO|.
```
While the enthusiasm is **always** appreciated 😄 we've been instructed to avoid using any exclamation points. Just gotta stick with boring periods, I'm afraid 😆

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:159]:
```suggestion
On the new |PO|, the :guilabel:`Order Deadline` field changes :guilabel:`Confirmation Date`, which
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:171]:
```suggestion
   With the **Inventory** app installed, confirming a |PO| automatically creates a receipt document,
```

**Review by ksc-odoo — CHANGES_REQUESTED** [review]
Hey there, @dikd-odoo -- just finished my Final Review of this PR - the first of **many** to come. Great job! Truly impressed with your RST skills straight-out-the-gate! You've created a very easy-to-follow piece of documentation, packed with useful information, that any reader would find super helpful.

As this is your first PR, there are a handful of suggestions/edits/feedback that require your attention, to help you get a better handle of how we approach the formatting/explaining of certain elements of the U/I. No worries, though. There's nothing major.

Feel free to take your time to review my comments, and implement the necessary changes. Then, once you feel like it's ready for another look, tag me again, and I'll happily give this "Pioneer PR" of yours another look.

But, seriously, this is a GREAT first PR! You're already crushing it! 👍 👏 

If you have any questions, comments, or concerns about anything, just let me know, and I'll be happy to help out.

Once again -- great job! 👏

**Comment by dikd-odoo** [comment]
Hi @ksc-odoo thank you so much for the kind words and final review! I've implemented your suggested changes (also noting the content feedback for future docs), this is ready for you take another look.

**Comment by dikd-odoo** [comment]
Hi @ksc-odoo thank you for the great feedback and kind words! I've implemented your suggested changes (noting the content feedback for future docs as well). Had some version control issues because I was switching between branches/docs but I think I resolved them; let me know if there's anything weird that comes up in your next look.

Thank you again!

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:75]:
```suggestion
:guilabel:`To Send`: orders in the |RFQ| stage that have not been sent to the vendor.
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:112]:
```suggestion
and vendor lead time. Tick the checkbox for :guilabel:`Ask confirmation` to ask for signage at
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:127]:
```suggestion
In the :guilabel:`Products` tab, add the products to be ordered. Click :guilabel:`Add a product`,
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:131]:
```suggestion
select :guilabel:`Create [product name]` from the resulting drop-down menu, and manually add the
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:163]:
```suggestion
   Messages, notes, and activities can also be logged on the chatter.
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:167]:
```suggestion
On the new |PO|, the :guilabel:`Order Deadline` field changes to :guilabel:`Confirmation Date`, which
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:172]: It could be beneficial to the reader to use the actual title of the Documentation that's being referenced here, just so they can also locate it on their own, if needed. This is just a suggestion, though. Not a requirement. Just something to consider during your revisions. 👍

**Review by ksc-odoo — APPROVED** [review]
hey @dikd-odoo -- just gave this another look. Awesome job implementing the previous round of feedback. Everything's looking great. There are only a small handful of super-minor adjustments that require your attention. But, I am approving now, because I don't anticipate you having any issues with them. Once you take care of those, feel free to tag this for Tech Review. Thanks! 👍

**Comment by dikd-odoo** [comment]
Thanks @ksc-odoo! 

Hi @samueljlieber, this PR is ready for tech review!

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq/product-vendor-pricelist-config.png
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq/product-vendor-pricelist-config.png:1]: This image is a bit too wide at 1178px, please resize to be at or under 933px wide :)
p.s. dont forget to compress with pngquant after resizing!

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq/RFQ-dashboard.png
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq/RFQ-dashboard.png:1]: Please rename this image to be all lowercase, and then compress this image with pngquant :)

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:79]: Please use all lowercase in filenames :)
```suggestion
.. image:: rfq/rfq-dashboard.png
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:81]: Please no formatting (shorthand, italic, bold, guilabel, etc) in alt tags :)
```suggestion
   :alt: RFQ dashboard with orders and order statuses.
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:5]: Small change to this custom anchor, for convention's sake–

- Update the app/module to Purchase, not Inventory
- Can abbreviate to "RFQ" to make the anchor shorter


```suggestion
.. _purchase/manage_deals/rfq:
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:13]: Remove curly quote (`’`), these pesky quotes can be carried over from copy/pasting from a Google doc or other formatted text editor.

Also there is an early line break in this sentence, can break the line closer to 100 chars :)
```suggestion
Odoo's requests for quotation (RFQs) feature in the **Purchase** app standardizes ordering products
from multiple vendors with varying prices and delivery times.
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:30]: Remove double "the" :)
```suggestion
name. Then, go to the :guilabel:`Inventory` tab, and enable the :guilabel:`Buy` route.
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:66]: Remove smart quote
```suggestion
The :guilabel:`Requests for Quotation` dashboard displays an overview of the company's |RFQs|,
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:70]: Remove smart quote
```suggestion
The top-right corner also provides a quick report of the company's recent purchases by total value,
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:85]: Consider adding a seealso to this documentation! :)
```suggestion
.. seealso::
   :doc:`../../../essentials/search`

```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:141]: Text in an admonition needs to align with the first letter of the admonitions name, `n` in `.. note::` in this case :)
```suggestion
.. note::
   To make adjustments to products and prices, access the product form by clicking the
   :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon that becomes available upon hovering over
   the :guilabel:`Product` name.
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:168]: Need to break this line a little earlier, as "which" ends up being over 100 chars.
```suggestion
On the new |PO|, the :guilabel:`Order Deadline` field changes to :guilabel:`Confirmation Date`,
which displays the date and time the user confirmed the order.
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @dikd-odoo! Awesome work on this brand new Purchase doc on RFQs. This PR is looking really great, I only have a small handful of technical changes! Please see below, and tag me for another look once you have had a chance to address them.

Great work on your first documentation PR! 👏 🎉

Thank you for your work! and let me know if you have any questions!

**Comment by dikd-odoo** [comment]
Hi @samueljlieber thank you for the great feedback! I've incorporated your changes; this PR is ready for another look when you have a chance. :)

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:47]: Small change to update the icons label name, this icon is usually referred to as "additional options" :)
```suggestion
To enable or disable columns, click the :icon:`oi-settings-adjust` :guilabel:`(additional options)`
icon on the right side of the header row to reveal a drop-down menu of additional columns that can
be added (or removed) from the :guilabel:`Purchase` tab.
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst:79]: This seems like it should be formatted as a list, feel free to ignore!
```suggestion
- :guilabel:`To Send`: orders in the |RFQ| stage that have not been sent to the vendor.
- :guilabel:`Waiting`: |RFQs| that have been sent by email, and are waiting on vendor confirmation.
- :guilabel:`Late`: |RFQs| or |POs| where the :guilabel:`Order Deadline` has passed.
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/rfq/product-vendor-pricelist-config.png
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/manage_deals/rfq/product-vendor-pricelist-config.png:1]: _no change needed_
Couple notes for future reference (using this image as an example):
- we are no longer marking up images
- this image could be taken with more zoom, text is a bit small and there is a lot of empty space

**Review by samueljlieber — APPROVED** [review]
Hi @dikd-odoo! Thanks for implementing my changes! Everything is looking great :)

I am approving with one needed change, a suggestion, and then a little FYI! 

Please implement the needed change, and consider the suggestion. Then feel free to merge after all checks pass. Thank you!

@robodoo delegate=dikd-odoo

**Comment by dikd-odoo** [comment]
@robodoo r+

**Comment by dikd-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
